website/blog/2018-12-19-webdriverio-v5-released.md

```diff
@@ -0,0 +1,111 @@
+
+## New Package Structure
+
+As we moved to a monolithic system we scoped all WebdriverIO packages into the [`@wdio`](https://www.npmjs.com/org/wdio) NPM organization. This would make it simpler to onboard contributors to release new package versions and clarifies better which packages are "officially" maintained by the organization or are 3rd party community packages.<br>
+<br>
+If you have been using `wdio-mocha-framework` or `wdio-spec-reporter` in your project please update the packages to use the ones build for v5: `@wdio/mocha-framework` or `@wdio/spec-reporter`. The version number of all packages are from now on pinned to each other meaning that you should always have the same versions for all "official" WebdriverIO packages you use.
+
+## We Did Some Spring Cleaning
+
+When people proposed new commands to the API we contributors became more and more hesitent to introduce them due to the huge amount of commands that already existed. Many of these commands didn't provide any much differences from each other and maintaining them became a nightmare.<br>
+<br>
+Starting with v5 we created a "base" WebdriverIO package called [`webdriver`](https://www.npmjs.com/package/webdriver). It contains the bare logic to make HTTP request to a WebDriver endpoint and has all commands from [various specification](https://github.com/webdriverio/webdriverio/tree/master/packages/webdriver/protocol) (including the [WebDriver](https://w3c.github.io/webdriver/) spec as well as Appiums [Mobile JSONWire protocol](https://github.com/SeleniumHQ/mobile-spec/blob/master/spec-draft.md)) defined in a simple to maintain JSON object.<br>
+<br>
+As part of that effort we've renamed a lot of commands to align them to the pattern that has been used in the protocol. The base WebDriver client also now returns the `value` property of all protocol commands responses so that we were able to get rid of a lot of duplication (e.g. `title` and `getTitle` is now called `getCurrentTitle`).<br>
+<br>
+Additionally we have seen a lot of confusion around the WebdriverIO API and the fact that it sometimes takes selectors as first argument and sometimes not which made it really hard to e.g. build a consistent and useful TypeScript definition for it. In the new version __we got rid of selectors as command parameters__ and defined a set of commands only callable when using the browser/client instance or when using an element instance. In order to click on an element you now need to first fetch it to receive an element object and then call the click command on that instance, e.g.
+
+```js
+browser.click('#elem') // throws 'browser.click is not a function'
+
+const elem = $('#elem')
+elem.click() // clicks successfully on the element
```

Comment:
$('#elem').click() -> is this , also, possible?

Reply:
yes